README.md

```diff
@@ -199,6 +199,10 @@ If an entry exists in `drive.version` of the `folder` but not in `version`, then
 
 Downloads all the blobs in `folder` corresponding to entries in `drive.checkout(version)` that are not in `drive.version`. In other words, downloads all the blobs added to `folder` up to `version` of the drive.
 
+#### `await drive.downloadRange(dbRanges, blobRanges)`
```

Comment:
Was moving this intended?

Reply:
Yep, it seemed like some sections had gotten out of order so I rearranged them 👍